src/Sdk/Sdk.csproj

```diff
@@ -8,7 +8,7 @@
         <NoWarn>NU1701;NU1603</NoWarn>
         <Version>$(Version)</Version>
         <DefineConstants>TRACE</DefineConstants>
-        <LangVersion>7.3</LangVersion>
+        <LangVersion>8.0</LangVersion>
```

Comment:
do you have to bump this?

Reply:
Without bumping, we can't use Nullable Reference Types, which would mean that we can't just copy the deserialization for the actions resolution response from actions-dotnet